layouts/404.html

```diff
@@ -0,0 +1,11 @@
+{{ define "head-title" }}
```

Comment:
Thanks for clarifying. Based on the directions you linked to, we may need to configure Apache to display this page. We can certainly deploy this, then adjust as necessary.